docs/install.md

```diff
@@ -15,8 +15,8 @@
     * `Service Account User`: Used when attaching disks
   * Create a new key for your service account, and then point to it with an environment variable:
     * `export GOOGLE_APPLICATION_CREDENTIALS="/home/foo/service_account_creds.json"`
-  * Alternately you can run Turbinia under your own credentials (not recommended).  Run 'gcloud auth login' (may require you to copy/paste url to browser).
-  * Or run 'gcloud auth application-default login'
+  * Alternately you can run Turbinia under your own credentials (not recommended).  Run `gcloud auth login` (may require you to copy/paste url to browser).
+  * Or run `gcloud auth application-default login`
```

Comment:
For some reason it seems to be rendering these changes incorrectly (no newline and literal '*' instead of a bullet).   Can you fix the indenting here?

Reply:
Hello:  Just wanted to check on this.  Please let me know if you want to adjust this PR.  If I don't hear from you in a few days, I may just recreate it so we can get it submitted.  Thanks!